config.js

```diff
@@ -369,6 +369,9 @@ config.on_premise = {
     nva_part: "NVA_Upgrade.tgz"
 };
 
+// the threshold in ms for logging long running queries
+config.DB_LOG_QUERY_THRESHOLD = 1000;
```

Comment:
Maybe this should be bigger? Did we check how much queries are bigger than 1 sec? I'm pretty sure that in minikube it will print a lot, not sure if in normal clusters.... So maybe we should disable for minikube or maybe set specific for it something bigger?

Reply:
I added an env to have a way to change it easily. 
I think that the majority of queries should not take longer than 1 sec. it is valuable to know of queries that do take longer than that, so we can fix them if needed. we can tune it later if we see it overloads the logs.
WDYT?